File: Python Fundamentals/Data Types and Variables/gladiator_expenses.py
lost_fights = int(input())
helmet_price = float(input())
sword_price = float(input())
shield_price = float(input())
armor_price = float(input())
expenses = 0
shield_brakes = 0
for i in range(1, lost_fights + 1):
    if i % 2 == 0:
        expenses += helmet_price
    if i % 3 == 0:
        expenses += sword_price
        if i % 2 == 0:
            expenses += shield_price
            shield_brakes += 1
            if shield_brakes == 2:
                expenses += armor_price
                shield_brakes = 0

print(f"Gladiator expenses: {expenses:.2f} aureus")

File: JavaScript Fundamentals/JavaScript Fundamentals Data Types/gladiatorExpenses.js
function gladiatorExpenses(lostFights, helmetPrice, swordPrice, shieldPrice, armorPrice){
    let expenses = 0;
    shieldsBrakes = 0;
    for (let i = 1; i <= lostFights; i++){
        if (i % 2 === 0 && i % 3 === 0){
            expenses += helmetPrice + swordPrice + shieldPrice;
            shieldsBrakes += 1;
            if (shieldsBrakes % 2 === 0){
                expenses += armorPrice;
            }
        } else if(i % 2 === 0){
            expenses += helmetPrice;
        } else if(i % 3 === 0){
            expenses += swordPrice;
        } 
    }
    console.log(`Gladiator expenses: ${expenses.toFixed(2)} aureus`);
}

gladiatorExpenses(7,
    2,
    3,
    4,
    5);

gladiatorExpenses(23,
    12.50,
    21.50,
    40,
    200);